src/main/drivers/accgyro/accgyro_spi_icm42605.c

```diff
@@ -220,21 +222,27 @@ void icm42605GyroInit(gyroDev_t *gyro)
     }
 
     uint8_t value;
+#ifndef DEBUG_ICM42605_REGISTERS
+    UNUSED(value);
+#endif
 
     STATIC_ASSERT(INV_FSR_2000DPS == 3, "INV_FSR_2000DPS must be 3 to generate correct value");
     spiBusWriteRegister(&gyro->bus, ICM42605_RA_GYRO_CONFIG0, (3 - INV_FSR_2000DPS) << 5 | (outputDataRate & 0x0F));
     delay(15);
 
     value = spiBusReadRegister(&gyro->bus, ICM42605_RA_GYRO_CONFIG0);
+#ifdef DEBUG_ICM42605_REGISTERS
```

Comment:
Directly writing into `debug[]` is not supported any more, as it interferes with runtime selection of the debug mode - this code should be removed, or else turned into code using `DEBUG_SET()` - but since the debug output is trivial it's probably easiest to remove it.

Reply:
agreed, commit added to remove the reads and the `debug` usage entirely.